hack/karmada-bootstrap.sh

```diff
@@ -20,21 +21,45 @@ MEMBER_CLUSTER_KUBECONFIG=${MEMBER_CLUSTER_KUBECONFIG:-"${KUBECONFIG_PATH}/membe
 MEMBER_CLUSTER_1_NAME=${MEMBER_CLUSTER_1_NAME:-"member1"}
 MEMBER_CLUSTER_2_NAME=${MEMBER_CLUSTER_2_NAME:-"member2"}
 PULL_MODE_CLUSTER_NAME=${PULL_MODE_CLUSTER_NAME:-"member3"}
-CLUSTER_VERSION=${CLUSTER_VERSION:-"kindest/node:v1.19.1"}
+HOST_IPADDRESS=${1:-}
 
+CLUSTER_VERSION=${CLUSTER_VERSION:-"kindest/node:v1.19.1"}
 KIND_LOG_FILE=${KIND_LOG_FILE:-"/tmp/karmada"}
 
 #step0: prepare
+# Make sure go exists
+util::cmd_must_exist "go"
 # install kind and kubectl
 util::install_tools sigs.k8s.io/kind v0.10.0
+# get arch name and os name in bootstrap
+BS_ARCH=$(go env GOARCH)
```

Comment:
Does the "go env GOARCH" cmd only return "amd64" or "arm64"? If it return other values, maybe we need to do a check. See https://golang.org/doc/install/source#environment

Reply:
I didn't get your point.
Do you mean we should stop running here if not mac nor linux?

Reply:
> I didn't get your point.
> Do you mean we should stop running here if not mac nor linux?

The variable "BS_ARCH" will be part of kubectl download url, like "https://dl.k8s.io/release/"$KUBECTL_VERSION"/bin/linux/"$ARCH"/kubectl". If return value is not amd64 or arm64, we can't download the kubectl binary.

Reply:
Yes, indeed. I'll add a filter for GOARCH and GOOS